tests/lib/snapshots/markdown-test.js.md

```diff
@@ -226,39 +226,4 @@ Generated by [AVA](https://ava.li).
 > Snapshot 2
 
     `<p data-line="0">This is a &quot;QUOTE&quot;.</p>␊
-    
-
-## Markdown.render() should render PlantUML Ditaa correctly
```

Comment:
Why do you have to remove these tests?

Reply:
I just deleted the snapshots, to regenerate it. Do I need to it in another way?

Reply:
That's strange, somehow it's still pass the test

Reply:
How did you deleted them?

Reply:
a normal delete, direct from the IDE

Reply:
Oh they have been moved to the top of the file :smile: look good to me then :smile:

Reply:
heheh troll